test/unit/api-client/api-client.submit.spec.ts

```diff
@@ -112,6 +137,34 @@ describe('api-client job submission', () => {
             expect(job).toEqual(jobDetails);
         });
 
+        it('submit job with null options', async () => {
+            const mockHandler = ApiRequestHandler.mock.instances[0];
+            mockHandler.makeApiRequest.mockResolvedValue(jobDetails);
+            let fakeStream = new Buffer(10);
+            const options = {
+                metadata: null,
+                callback_url: null,
+                custom_vocabularies: null,
+                skip_punctuation: null,
+                skip_diarization: null,
+                speaker_channels_count: null,
+                filter_profanity: null
+            };
+
+            const job = await sut.submitJobAudioData(fakeStream, null, options);
+
+            const expectedPayload = expect.objectContaining({
+                '_boundary': expect.anything(),
+                '_streams': expect.arrayContaining([expect.anything(),
+                    expect.stringContaining('Content-Disposition: form-data; name="media"')])
+            });
+            const expectedHeader = { 'content-type': expect.stringMatching(/multipart\/form-data; boundary=.+/) };
+            expect(mockHandler.makeApiRequest).toBeCalledWith('post', '/jobs',
```

Comment:
There are no new test assertions in this test.

Either add a relevant assertion or remove this  test.

Additionally, the test case name `submit job with null options` isn't accurate. I think

Reply:
You should assert that these properties are not present: https://github.com/revdotcom/revai-node-sdk/pull/41/files#diff-55616ed1583fc5edff382eb7d3ddcc36R189-R197

Reply:
Addressed this in all async client tests testing for `all options null`.

Note that in [this test](https://github.com/revdotcom/revai-node-sdk/blob/feature/AISDK-92/test/unit/api-client/api-client.submit.spec.ts#L51-L70) the check is also made despite not being in the same pattern as the other null options tests.